docs/contents/components/forms/dropzone/index.mdx

```diff
@@ -219,10 +249,24 @@ Please note that it is always called regardless of whether the dropped file is a
     </DropzoneIdle>
 
     <VStack gap="2xs">
-      <Text fontSize="xl">Drag images here or click to select files</Text>
-      <Text fontSize="sm">
-        Attach as many files as you like, each file should not exceed 3mb
-      </Text>
+      <For
```

Comment:
There is no need to use `For` here. I feel that the modification makes it less readable.

Reply:
Fixed.